person.rb

```diff
@@ -1,4 +1,9 @@
 # class representing a person
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
 class Person
```

Comment:
- [ ] **[OPTIONAL]** This is what your person initializer looks like. But it should be this instead. The `age` parameter should come before the `name` parameter and it should not be optional

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
  ```

  ![image](https://github.com/IvonneBenitesRodriguez/oop-school-library/assets/36542986/cf47c9c1-0e3d-441c-8e5e-61a65acbc797)

Reply:
Dear @TedLivist,
Good morning
Very nice to meet you, 
Thank you so much for your understanding and support, 
I truly appreciate it, 
I applied now the changes you suggested, I want to say that you
helped me to understand why I got my first mistake, but now i see that
order matters in Ruby, I mean for the position of name, and age,
Thanks again, 
Best wishes,